src/lib/fast_check.py

```diff
@@ -213,6 +208,18 @@ async def execute(self) -> FastCheckResult:
                                   dynatrace_url=dynatrace_url,
                                   dynatrace_access_key=dynatrace_access_key)
 
+    async def execute(self) -> FastCheckResult:
+        _check_configuration_flags(self.logging_context, METRICS_CONFIGURATION_FLAGS)
+
+        project_list = await get_all_accessible_projects(self.logging_context, self.gcp_session, self.token)
+
+        ready_to_monitor = []
+        tasks_to_find_ready_projects = []
+        for project_id in project_list:
+            tasks_to_find_ready_projects.append(self.is_project_ready_to_monitor(project_id, ready_to_monitor))
```

Comment:
is async routine adding stuff to an array that is here (level above)? This is a little messy design, is this much work to have async routine return result like (<PROJECT_ID>, <CAN_MONITOR=TRUE/FALSE>) and only take IDs from the results that we get from asyncio gather?
Side effects (modifying external objects) should be limited to minimum where possible